README-devel.md

```diff
@@ -106,6 +109,22 @@ $ cosa run --ro-bind /usr/bin,/run/hostbin
 Then in the VM you have e.g. `/run/hostbin/strace`.  (This may fail in some scenarios
 where your dev container is different than the target).
 
+# Using cosa buildinitramfs-fast
+
+If you're iterating on changes *just* to the initramfs, you can also use
+`cosa buildinitramfs-fast`.  For example, suppose you are working on `ignition`.
+Follow these steps:
+
+```
+$ make
+$ install -D -m 0755 bin/amd64/ignition /path/to/cosadir/overrides/initramfs/usr/bin/ignition
+$ cd /path/to/cosadir
+$ cosa buildinitramfs-fast
+$ cosa run --qemu-image tmp/fastbuild/fastbuildinitrd-fedora-coreos-qemu.qcow2
+```
+
+(Or instead of `cosa run` use e.g. `cosa kola` to run tests, etc.)
```

Comment:
A little bit confused by this line.. This is the step following the overriding step, right?

Reply:
It's saying that instead of the last step (`cosa run`) one can use `cosa kola` - do you have any alternative suggestions?

Reply:
Ahh I misunderstood during first read.. marking as resolved